README.md

```diff
@@ -1,11 +1,11 @@
 Indexd
 ===
-![version](https://img.shields.io/badge/version-0.0.1-orange.svg?style=flat) [![Apache license](http://img.shields.io/badge/license-Apache-blue.svg?style=flat)](LICENSE) [![Travis](https://travis-ci.org/LabAdvComp/indexd.svg?branch=master)](https://travis-ci.org/LabAdvComp/indexd)
+![version](https://img.shields.io/badge/version-0.0.1-orange.svg?style=flat) [![Apache license](http://img.shields.io/badge/license-Apache-blue.svg?style=flat)](LICENSE) [![Travis](https://travis-ci.org/uc-cdis/indexd.svg?branch=master)](https://travis-ci.org/uc-cdis/indexd)
 
 Indexd is a prototype data indexing and tracking service. It is intended to be
 distributed, hash-based indexing service, designed to be accessed via a
 REST-like API or via a client, such as the
-[reference implementation](https://github.com/LabAdvComp/index).
+[reference implementation](https://github.com/uc-cdis/index).
```

Comment:
Missing a 'd'

Reply:
oops, that's actually `indexclient` which was previously called `index`, fixed